src/components/AddChecklistForm/AddChecklistForm.jsx

```diff
@@ -5,20 +5,20 @@ import styles from './AddChecklistForm.module.css'
 function AddChecklistForm(props) {
   const [checklistData, setChecklistData] = useState({
     // owner: '',
-    name: '',
+    checklistName: '',
     items: [''],
   })
 
   const handleChange = (e) => {
-    setChecklistData({...checklistData, [e.target.name]: e.target.value})
+    setChecklistData({...checklistData, [e.target.checklistName]: e.target.value})
```

Comment:
@vzarate-94 I don't think this is correct.

`e.target.name` refers to an attribute of the form input.